components/bm1397/bm1397.c

```diff
@@ -154,7 +154,7 @@ void BM1397_send_hash_frequency(float frequency)
     fb = 2;
     fc1 = 1;
     fc2 = 5; // initial multiplier of 10
-    if (f1 >= 500)
+    if (f1 > 650)
```

Comment:
i think we should keep the >=500 here, to halve down from [500-650] to [250-325]